cloudwash/providers/ec2.py

```diff
@@ -42,6 +42,11 @@ def dry_pips():
             [dry_data["PIPS"]["delete"].append(dpip["AllocationId"]) for dpip in rpips]
             return dry_data["PIPS"]["delete"]
 
+        def dry_s3():
+            s3_buckets = ec2_client.list_s3_bucket_names()
```

Comment:
This is fetching the list of all the S3 buckets not something which are not in use. Due to this, we will end up deleting all the S3 buckets including those which we dont to delete.
 
So we need to decide the criteria for deleting the buckets.
It could be similar to VM - like buckets running from some_x_time/ empty / prefix_string names etc.

Reply:
If you still want to stick to removing all the s3 buckets, the options should clearly mention that and definitions in this PR should be renamed to `all_s3`.